### PR TITLE
refactor(gio-table-light): mv typo to component

### PR DESCRIPTION
**Pull request by gcusnieux**
**Issue**

https://github.com/gravitee-io/issues/issues/6765

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://6183b02d73381a003a3be1a6-xofehaphcd.chromatic.com)

### CODE REVIEW

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

A prerelease version of this package has been published! 🚀

To use it in your project run the following lines:
```shell
npm install @gravitee/ui-particles-angular@2.4.0-debug-add-inspecto-53d340a
# or
yarn add @gravitee/ui-particles-angular@2.4.0-debug-add-inspecto-53d340a
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

A prerelease version of this package has been published! 🚀

To use it in your project run the following lines:
```shell
npm install @gravitee/ui-policy-studio-angular@2.4.0-debug-add-inspecto-53d340a
# or
yarn add @gravitee/ui-policy-studio-angular@2.4.0-debug-add-inspecto-53d340a
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

A prerelease version of this package has been published! 🚀

To use it in your project run the following lines:
```shell
npm install @gravitee/ui-policy-studio-angular@2.4.0-debug-add-inspecto-76d61d2
# or
yarn add @gravitee/ui-policy-studio-angular@2.4.0-debug-add-inspecto-76d61d2
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

A prerelease version of this package has been published! 🚀

To use it in your project run the following lines:
```shell
npm install @gravitee/ui-particles-angular@2.4.0-debug-add-inspecto-76d61d2
# or
yarn add @gravitee/ui-particles-angular@2.4.0-debug-add-inspecto-76d61d2
```

File: ui-particles-angular/projects/ui-particles-angular/src/scss/theme/typography/gio-typography.scss [hunk 1]
```diff
@@ -13,10 +13,12 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-@mixin subtitle-typography() {
+@mixin body-strong-typography() {
```
Comment (ThibaudAV) [inline ui-particles-angular/projects/ui-particles-angular/src/scss/theme/typography/gio-typography.scss:16]: normally this still exist : 
image

File: ui-particles-angular/projects/ui-particles-angular/src/scss/theme/typography/gio-typography.scss [hunk 2]
```diff
@@ -47,3 +45,14 @@
   letter-spacing: 0.4px;
   text-align: left;
 }
+
+@mixin table-head-typography() {
```
Comment (ThibaudAV) [inline ui-particles-angular/projects/ui-particles-angular/src/scss/theme/typography/gio-typography.scss:49]: could we move all table stuff inside gio-table component ?

File: ui-particles-angular/projects/ui-particles-angular/src/scss/theme/typography/gio-typography.scss [hunk 3]
```diff
@@ -47,3 +45,14 @@
...
+  font-family: 'Inter', monospace;
```
Comment (ThibaudAV) [inline ui-particles-angular/projects/ui-particles-angular/src/scss/theme/typography/gio-typography.scss:50]: not needed, default Font-Family should be used 🤔

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

A prerelease version of this package has been published! 🚀

To use it in your project run the following lines:
```shell
npm install @gravitee/ui-particles-angular@2.4.0-debug-add-inspecto-c422667
# or
yarn add @gravitee/ui-particles-angular@2.4.0-debug-add-inspecto-c422667
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

A prerelease version of this package has been published! 🚀

To use it in your project run the following lines:
```shell
npm install @gravitee/ui-policy-studio-angular@2.4.0-debug-add-inspecto-c422667
# or
yarn add @gravitee/ui-policy-studio-angular@2.4.0-debug-add-inspecto-c422667
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

A prerelease version of this package has been published! 🚀

To use it in your project run the following lines:
```shell
npm install @gravitee/ui-policy-studio-angular@2.4.0-debug-add-inspecto-14ef5af
# or
yarn add @gravitee/ui-policy-studio-angular@2.4.0-debug-add-inspecto-14ef5af
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

A prerelease version of this package has been published! 🚀

To use it in your project run the following lines:
```shell
npm install @gravitee/ui-particles-angular@2.4.0-debug-add-inspecto-14ef5af
# or
yarn add @gravitee/ui-particles-angular@2.4.0-debug-add-inspecto-14ef5af
```

**Review by ThibaudAV — APPROVED** [review]

**Pull request merged**